src/webview/static/base.css

```diff
@@ -484,12 +484,13 @@ time {
 }
 
 .message_inline_image {
-  text-align: center;
+  margin-top: 5px;
```

Comment:
What's the reasoning for adding to the margin here? Is this related to the new left-alignment, or is it for another reason?

Reply:
No its not related to new alignment, rather it's just to make sure mobile has same styling design with web-app, since there it has been kept 
`margin:3px 0 3px 0`([here](https://github.com/zulip/zulip/blob/6b7a3fb74acb9e8992d256a2da3b75fcb22befb9/static/styles/rendered_markdown.css#L4)) and that provides necessary 3px gap between image link and image.(and since unlike web ,here image link underline remains as it is so it seemed better with 5px)